computer_vision/harriscorner.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import cv2
+
+"""
+Harris Corner Detector
+https://en.wikipedia.org/wiki/Harris_Corner_Detector
+"""
+
+
+class Harris_Corner:
+    def __init__(self, k, window_size):
```

Comment:
Type hints?  What is `k`?  Could we have a more self documenting variable name?

Reply:
it's a constant, also represented the same in wiki page